src/service.ui/index.js

```diff
@@ -198,10 +228,12 @@ export default class UIService {
    @returns { void }
    */
   setTheme = (config = {}) => {
-    const darkMode = config?.darkMode || vuelessConfig?.darkMode || DEFAULT_DARK_MODE;
-    const rounding = config?.rounding || vuelessConfig?.rounding || DEFAULT_ROUNDING;
     const brand = config?.brand || vuelessConfig?.brand || DEFAULT_BRAND_COLOR;
     const gray = config?.gray || vuelessConfig?.gray || DEFAULT_GRAY_COLOR;
+    const ring = config?.ring || vuelessConfig?.ring || DEFAULT_RING;
+    const ringOffset = config?.ringOffset || vuelessConfig?.ringOffset || DEFAULT_RING_OFFSET;
+    const rounding = config?.rounding || vuelessConfig?.rounding || DEFAULT_ROUNDING;
+    const darkMode = config?.darkMode || vuelessConfig?.darkMode || DEFAULT_DARK_MODE;
```

Comment:
As in previous MR, probably we should check for user preferences:
_"We probably should check if darkMode !== undefined here. If darkMode is undefined, we will look in localStoarage and window.matchMedia("(prefers-color-scheme: dark)"); to set theme preferred by user or browser."_